packages/dappmanager/src/modules/chains/getChainDriverName.ts

```diff
@@ -19,8 +19,8 @@ const knownChains: { [dnpName: string]: ChainDriver } = {
   "monero.dnp.dappnode.eth": "monero",
   "prysm.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
   "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
-  "teku.dnp.dappnode.eth": "ethereum2",
-  "teku-prater.dnp.dappnode.eth": "ethereum2",
-  "lighthouse.dnp.dappnode.eth": "ethereum2",
-  "lighthouse-prater.dnp.dappnode.eth": "ethereum2"
+  "teku.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "teku-prater.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "lighthouse.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "lighthouse-prater.dnp.dappnode.eth": "ethereum-beacon-chain"
```

Comment:
Remove this from the defaults. This packages have not been published so we should only add defaults when really necessary